packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/marker.dart

```diff
@@ -64,7 +64,8 @@ class InfoWindow {
     );
   }
 
-  Object _toJson() {
+  /// Converts this object to something serializable in JSON.
+  Object toJson() {
```

Comment:
This object only has three fields and they are all simple; it would be much easier to include it in this Pigeon conversion than to make this a multi-package PR. (Also, exposing more of the JSON surface publicly is not desirable, as this is all legacy code that we are going to want to remove someday.)